manifests/weave/boarnet/bns/genesis.json

```diff
@@ -0,0 +1,458 @@
+        "//name": "Ingrid",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "0EDAA1225975A3A64BC7216F9429A3EA34A757D9",
+        "//name": "Jonny",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+        "//name": "Cloe",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "cond:gov/rule/0000000000000002",
+        "//name": "reward fund",
+        "coins": [
+          {
+            "whole": 1,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+        "//name": "faucet",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "B1DE2F431945B8D7CE20DDD5F43FDA9609E9D7B8",
+        "//name": "loadtester",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      }
+    ],
+    "currencies": [
+      {
+        "ticker": "IOV",
+        "name": "Main token of this chain"
+      }
+    ],
+    "multisig": [
+      {
+        "activation_threshold": 1,
+        "admin_threshold": 1,
+        "participants": [
+          {
+            "signature": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+            "//name": "Cloe",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+    "update_validators": {
+      "addresses": [
+        "cond:gov/rule/0000000000000004"
+      ]
+    },
+    "distribution": [
+      {
+        "//name": "unused - owned by economic committee",
+        "admin": "cond:gov/rule/0000000000000002",
+        "destinations": [
+          {
+            "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+
+    "escrow": [
+      {
+        "//name": "guarantee fund",
+        "amount": [
+          {
+            "ticker": "IOV",
+            "whole": 1000000
+          }
+        ],
+        "arbiter": "cond:gov/rule/0000000000000002",
+        "destination": "cond:gov/rule/0000000000000002",
+        "//destination": "reward fund - cash account",
+        "source": "0000000000000000000000000000000000000000",
+        "timeout": "2999-12-31T00:00:00Z"
+      },
+      {
+        "//name": "vesting first tranche",
+        "amount": [
+          {
+            "ticker": "IOV",
+            "whole": 1234567
+          }
+        ],
+        "arbiter": "0000000000000000000000000000000000000000",
+        "destination": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17",
+        "source": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17",
+        "timeout": "2019-08-28T00:00:00Z"
+      }
+    ],
+    "conf": {
+      "username": {
+        "owner": "cond:gov/rule/0000000000000001",
```

Comment:
q: `"cond:gov/rule/0000000000000001"` is the governing board. I just want to double check that it is not the `Tech Committee` instead? @davepuchyr

Reply:
I don't have a strong opinion on the `owner`.  It would be the governing board that might relax the constraints, though, so that's why I chose it.  They could, of course, inform the `Tech Committee` of the change.  I was thinking that there's less friction if only one committee is involved.